src/components/calendar/DmCalendarMessage.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="text-center">
```

Comment:
May the _text-center_ class be part of the `v-card`?

not very semantic the `div` tag